R/MaximizeParsimony.R

```diff
@@ -999,17 +999,13 @@ Resample <- function(dataset, tree, method = "jack", proportion = 2 / 3,
 #' Launch tree search graphical user interface
 #' 
 #' @rdname MaximizeParsimony
-#' @importFrom cluster pam silhouette
-#' @importFrom future future
```

Comment:
Is it really safe to just remove this entirely? How so?

There are a lot of packages removed here without an obvious fallback.